### PR TITLE
Leaf/blake/identify broncodirect page

### PR DESCRIPTION
**Pull request by dal07065**
Able to identify whether the current page is on BroncoDirect and identify whether the specific state of the page represent the Search Criteria page or Search Results Page

### CODE REVIEW

**Comment by alimomennasab** [comment]
Looks good, cleaned up all of the excess comments and fixed the merging issues.

**Comment by alimomennasab** [comment]
Looks good, cleaned up the excess comments and fixed the merging errors. (Accidentally closed the pr earlier, sorry!)

**Review by alimomennasab — COMMENTED** [review]
Looks good, cleaned up the excess comments and fixed the merging errors.

**Review by alimomennasab — APPROVED** [review]
Looks good, cleaned up the excess comments and fixed the merging errors.

File: src/Content.tsx [hunk 1]
```diff
@@ -2,27 +2,39 @@ import React from 'react';
...
-    root.setAttribute('id', 'root');
-    root.style.float = 'right';
-    parentElem?.append(root);
+  // event listener on the iframe which fires on search
+  // since iframe will receive a post message of the search criteria
+  // however event will fire regardless if the search criteria is invalid and will remain on the page
+  iframe?.contentWindow?.addEventListener('message', () => {
+    const insts: NodeListOf<HTMLElement> | undefined =
+      iframe.contentWindow?.document.querySelectorAll('*[id^="MTG_INSTR$"]');
+    if (insts?.length == 0) console.log('- Enter Search Criteria Page');
```
Comment (Leaversa) [inline src/Content.tsx:21]:
```suggestion
    if (insts?.length === 0) console.log('- Enter Search Criteria Page');
```
it should be triple equals for the linter.

**Review by Leaversa — CHANGES_REQUESTED** [review]
I think you should consider using the header element text to determine which page is loaded rather than using the instructor elements. Since the header is always loaded on each page of BroncoDirect you're on you won't need to worry about bugs like if the elements doesn't load for whatever reason.

Here's some example code I wrote out that will get the title of the current page using this header method.
```
const iframe = document.getElementById('ptifrmtgtframe')
let header = iframe.contentWindow.document.querySelector('.gh-page-header-headings')
header.children[2].innerText
```

Other than that see my other comment about fixing the linting and your code should be good.

**Review by dal07065 — COMMENTED** [review]
Modified to use the header instead of the insts, professors list

**Review by dal07065 — COMMENTED** [review]
Reviewed

**Review by dal07065 — COMMENTED** [review]
Reviewed

**Review by GuyWhoCode — APPROVED** [review]

**Review by Leaversa — APPROVED** [review]

**Pull request merged**